syntax = "proto3";

package helloworld;

// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  rpc SayName (NameRequest) returns (NameReply) {}
}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings.
message HelloReply {
  string message = 1;
}


// The request message containing the user's name.
message NameRequest {
  string name = 1;
}

// The response message containing the greetings.
message NameReply {
  string message = 1;
}
